test/helpers.ts

```diff
@@ -0,0 +1,184 @@
+import {
```

Comment:
can we decompose this into respective classes instead of a `utlis` or `helpers` all-in-one bag?

For example a method like `createComponentItem` belongs to a class like `ComponentCreator` or `ComponentFactory`,  `createTagItem` to `TagFactory` etc ...

This way we will have a nicely decoupled and testable code. It's hard to maintain and test these kind of files.

Reply:
This file itself is a helper only for tests. Does your advice still apply @karolsojko?

Reply:
for tests it's ok